lib/cli/writer.js

```diff
@@ -23,7 +23,7 @@ const os = require('os');
 const unmount = require('./unmount');
 const imageStream = require('../image-stream');
 const errors = require('../shared/errors');
-const constraints = require('../shared/constraints');
+const constraints = require('../shared/drive-constraints');
```

Comment:
Even "only" running hardware tests on Linux, would still be better than no hardware tests at all? ;)